components/ui/popups.tsx

```diff
@@ -27,7 +27,8 @@ export enum PopupPlacement {
   BottomLeft = 'bottom-left',
   BottomRight = 'bottom-right',
   LeftTop = 'left-top',
-  RightTop = 'right-top'
+  RightTop = 'right-top',
+  TopLeft = 'top-left'
```

Comment:
I find this a bit confusing, what's the difference between `left-top` and `top-left`?
I guess renaming is a pain, but a jsdoc comment could help?

Reply:
Hi! I used the same logic than the other positioning, that is to say:

- **left-top** = the popup is positioned on the left side of the block, vertically aligned on top of the block;
- **top-left** = the popup is positioned above the block, horizontally aligned on the left of the block.

Not intuitive, I admit. I'll update the jsdoc.